src/applications/personalization/profile/components/proof-of-veteran-status/ProofOfVeteranStatus.jsx

```diff
@@ -91,12 +91,16 @@ const ProofOfVeteranStatus = ({
             <strong>Note: </strong>
             This card doesn’t entitle you to any VA benefits.
           </p>
-          <div className="vads-u-margin--1 vads-u-margin-y--2">
-            <img
-              width="400rem"
-              src="/img/proof-of-veteran-status-card-example.png"
-              alt="sample proof of veteran status card featuring name, date of birth, disability rating and period of service"
-            />
+          <div className="vads-l-grid-container vads-u-padding-y--2 vads-u-padding-l--0">
```

Comment:
It was driving me crazy why there was such a large margin at the left of the image. If we swap the `vads-l-grid-container` class out for `vads-l-grid-container--full` that fixes it. Then we can remove the `vads-u-padding-l--0` (which wasn't doing anything anyway)